README.md

```diff
@@ -166,8 +166,11 @@ pip install .
 # The Mac build requires switching through a few different python versions
 maturin build --cargo-extra-args="--features=python" --release --strip
 
-# The linux build is automated through cross-compiling in a docker image
-docker run --rm -v $(pwd):/io ghcr.io/pyo3/maturin build --cargo-extra-args="--features=python" --release --strip
+# The linux build requires switching through different python versions and linux compatibility targets.
+# For example, to build for Python 3.10 and manylinux2010 compatibility:
+docker run --rm -v $(pwd):/io ghcr.io/pyo3/maturin:main build --features=python --release --strip --interpreter=python3.10 --compatibility=manylinux2010
```

Comment:
`manylinux2010` is going to be removed soon actually but that's ok for now, I'll update it later.
The weird part is that you now need to specify the python version while the previous image was doing all of them automatically (and the help text says it should). I'll check on the maturin repo if it's intended.

Reply:
Ok the docker image is actually using the unreleased yet new maturin version which has this behaviour changed for some reason.